docs/reference_guide.md

```diff
@@ -2700,6 +2701,25 @@ Attaching 1 probe...
 # bpftrace -e 'BEGIN { $x = 3; printf("%d\n", sizeof($x)); }'
 Attaching 1 probe...
 8
+
+
+## 23. `path()`: Returns full path referenced by struct path pointer in argument
+
+Syntax: `path(struct path *path)`
+# bpftrace  -e 'kfunc:filp_close { printf("%s\n", path(args->filp->f_path)); }'
+Attaching 1 probe...
+/proc/sys/net/ipv6/conf/eno2/disable_ipv6
+/proc/sys/net/ipv6/conf/eno2/use_tempaddr
+socket:[23276]
+/proc/sys/net/ipv6/conf/eno2/disable_ipv6
+socket:[17655]
+/sys/devices/pci0000:00/0000:00:1c.5/0000:04:00.1/net/eno2/type
+socket:[38745]
+/proc/sys/net/ipv6/conf/eno2/disable_ipv6
+
+# bpftrace  -e 'kretfunc:dentry_open { printf("%s\n", path(retval->f_path)); }'
+Attaching 1 probe...
+/dev/pts/1 -> /dev/pts/1
```

Comment:
This ends up in the example codeblock of sizeof(). Also this should probably go at the end as number 25 (instead of reusing number 23)

Reply:
right, I reformatted it, thanks